alpha/scripts/compatCheck.php

```diff
@@ -624,14 +624,19 @@ function compareResults($resultNew, $resultOld)
 
 	if (!$resultNew)
 	{
-		return array('failed to parse new XML');
+		return array('#ERROR - failed to parse new XML');
 	}
 
 	if (!$resultOld)
 	{
 		return array('failed to parse old XML');
 	}
 
+	if (isset($resultNew['result']) && isset($resultNew['result']['error']) && !isset($resultOld['result']['error']))
```

Comment:
No need to check isset(isset($resultNew['result']['error']))
If checking isset($resultNew['result']['error']) and isset($resultNew['result']['error']) is not set - no error or notice is raised.